Test/Helper/ConfigTest.php

```diff
@@ -196,6 +197,20 @@ public function testIsMerchantActiveReturnsFalseWhenNotFound(): void
         $this->assertEquals(false, $this->config->isMerchantActive(1));
     }
 
+    public function initSdkConfiguration(): void
```

Comment:
probably want to name this something more descriptive (`testInitSdkConfigurationXXX` akin to other methods here) and add doc comment describing the test